supplementary_style_guide/glossary_terms_conventions/general_conventions/t.adoc

```diff
@@ -130,6 +130,33 @@
 
 *See also*:
 
+[discrete]
+[[tls]]
+==== image:images/yes.png[yes] TLS (noun)
+*Description*: TLS is an initialism for Transport Layer Security. TLS is the successor to the Secure Sockets Layer (SSL) protocol. TLS is a cryptographic protocol that uses the Public Key Infrastructure (PKI) method to encrypt network traffic between two systems. PKI uses symmetric encryption during a TLS handshake process to authenticate the connection between two systems.
+
+Use TLS when referring to protocols that exchange cryptographics keys and secure network connections between two systems. Check for the latest version of the TLS protocol and, if necessary, contact an SME to verify the TLS version to record in product documentation.
```

Comment:
```suggestion
Use TLS when referring to protocols that exchange cryptographic keys and secure network connections between two systems. Check for the latest version of the TLS protocol and, if necessary, contact a subject matter expert to verify the TLS version to note in product documentation.
```
- Spell out `SME`
- Consider changing `record` to `note`. `Record` doesn't seem like the right word here.

Reply:
:heavy_check_mark: